### DISCUSSION TITLE
Google Cloud Loader

### DISCUSSION THREAD

**Discussion by marcelocbf** [MinimalCompact/thumbor#155] [Q&A]
Hi,

I'm trying to configure my instance to load images from a Google Cloud Bucket, in addition, to HTTPS, but I'm having a hard time finding a way.

I found this package https://github.com/Superbalist/thumbor-cloud-storage which doesn't seem to be maintained anymore, but since I found mentions of CLOUD_STORAGE_PROJECT_ID and CLOUD_STORAGE_BUCKET_ID in the documentation I was wondering if it supported out of the box, but after trying to configure it with the proper keys that server doesn't recognize my path of /some/file.png as a valid path.

Thank you,

**Comment by gingerlime** [comment]
Unfortunately I have no experience using Google cloud with Thumbor. I would suggest asking on the upstream [thumbor](https://github.com/thumbor/thumbor) project.

**Comment by marcelocbf** [comment]
Thank you @gingerlime. I will
